package.xml

```diff
@@ -122,21 +126,23 @@
         <delete  dir="${build.unpack.dir}" />
         <mkdir dir="${build.unpack.dir}" />
         <unzip src="${zip.path}" dest="${build.unpack.dir}"/>
-        <tar destfile="${gzip.path}" basedir="${build.unpack.dir}" compression="gzip" excludes="${release}/bin/**,${release}/samps/**,${release}/projects/**">
+        <tar destfile="${gzip.path}" basedir="${build.unpack.dir}" compression="gzip" excludes="${release}/bin/**,${release}/samps/**,${release}/projects/**,${release}/scripts/**">
             <tarfileset mode="755" dir="${build.unpack.dir}">
                 <include name="${release}/bin/**"/>
                 <include name="${release}/samps/scripts/**"/>
                 <include name="${release}/samps/sleep.exe"/>
                 <include name="${release}/samps/**/distribute_score**"/>
                 <include name="${release}/samps/**/rotate**"/>
-                <include name="${release}/sandbox/**"/>
+                <include name="${release}/scripts/*.sh"/>
+                <include name="${release}/scripts/*.cmd"/>
```

Comment:
Previously, "all" files in the (formerly-named "sandbox") folder were included (`sandbox/**`).  Now, only `.cmd` and `.sh` files are being included.  This excludes, for example, the file `README.txt` in that folder -- which has some useful information about the organization and purpose of the files in the folder.  Shouldn't `README.txt` be included?  In fact, is there any reason not to just use `scripts/**`, analogously to what was there before?